Checker.cpp

```diff
@@ -2,19 +2,33 @@
 #include <iostream>
 using namespace std;
 
-bool batteryIsOk(float temperature, float soc, float chargeRate) {
+bool isTemperatureOk(float temperature) {
   if(temperature < 0 || temperature > 45) {
     cout << "Temperature out of range!\n";
     return false;
-  } else if(soc < 20 || soc > 80) {
+  }
+  return true;
+}
+ 
+bool isSocOk(float soc) {
+  if(soc < 20 || soc > 80) {
     cout << "State of Charge out of range!\n";
     return false;
-  } else if(chargeRate > 0.8) {
+  }
+  return true;
+}
+ 
+bool isChargeRateOk(float chargeRate) {
+  if(chargeRate > 0.8) {
     cout << "Charge Rate out of range!\n";
```

Comment:
wrap cout statement in another function

Reply:
Another function implemented to print the error message